controller/volume_controller.go

```diff
@@ -267,6 +272,27 @@ func (vc *VolumeController) syncVolume(key string) (err error) {
 		log.Debugf("Volume got new owner %v", vc.controllerID)
 	}
 
+	// If the current node where this controller is running on doesn't have the default engine image deployed,
```

Comment:
Move this into the isResponsible check example in the above pr comment.

Reply:
Thank you. Fixed